python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -770,6 +772,12 @@ def run_optimize_model_gpu(repo_id,
         model = optimize_model(model, low_bit=low_bit)
         tokenizer = LlamaTokenizer.from_pretrained(model_path, trust_remote_code=True)
         model = model.to('xpu')
+    elif repo_id in LLAMA3_VISION_IDS:
```

Comment:
Let's remove import of `MllamaForConditionalGeneration` inside the `elif` condition, in case it is not supported by `transformers` with lower version.